Documentation/ceph-upgrade.md

```diff
@@ -356,60 +277,41 @@ These images are tagged in a few ways:
 * Ceph major version tags (e.g., `v13`) are useful for development and test clusters so that the
   latest version of Ceph is always available.
 
-**Ceph containers other than the official images above will not be supported.**
+**Ceph containers other than the official images from the registry above will not be supported.**
 
-### Example upgrade to Ceph Mimic
+### Example upgrade to Ceph Nautilus
 
 #### 1. Update the main Ceph daemons
 The majority of the upgrade will be handled by the Rook operator. Begin the upgrade by changing the
 Ceph image field in the cluster CRD (`spec:cephVersion:image`).
+<!--                  TODO: CEPH IMAGE BELOW NEEDS UPDATE -->
 ```sh
-# sed -i.bak "s%image: .*%image: $NEW_CEPH_IMAGE%" cluster.yaml
-# kubectl -n $ROOK_SYSTEM_NAMESPACE replace -f cluster.yaml
-NEW_CEPH_IMAGE='ceph/ceph:v13.2.2-20181023'
+NEW_CEPH_IMAGE='ceph/ceph:v14.2.0-20190410'
 CLUSTER_NAME="$ROOK_NAMESPACE"  # change if your cluster name is not the Rook namespace
 kubectl -n $ROOK_NAMESPACE patch CephCluster $CLUSTER_NAME --type=merge \
   -p "{\"spec\": {\"cephVersion\": {\"image\": \"$NEW_CEPH_IMAGE\"}}}"
 ```
 
-As with upgrading Rook, you must now [wait for the upgrade to complete](#5.-wait-for-the-upgrade-to-complete).
-Unlike with the Rook upgrade, there is no at-a-glance sign that the upgrade is complete. We
-suggest watching the cluster upgrade carefully, and it is likely safe to assume the upgrade is
-complete if there have been no pods terminated in over a minute.
-```sh
-watch -c kubectl -n $ROOK_NAMESPACE get pods
-```
-
-To verify the Ceph upgrade is complete, check that all the images Rook is using are the newest ones.
+#### 2. Wait for the daemon pod updates to complete
+As with upgrading Rook, you must now wait for the upgrade to complete. Determining when the Ceph
+version has fully updated is rather simple.
 ```sh
-kubectl -n $ROOK_NAMESPACE describe pods | grep "Image:.*ceph/ceph" | sort | uniq
+watch --color kubectl -n $ROOK_NAMESPACE describe pods | grep "Image:.*ceph/ceph" | sort | uniq
```

Comment:
This `watch` command hangs for me, but if I remove `watch --color`, the rest of the command works fine.